modules/nf-core/flye/main.nf

```diff
@@ -1,6 +1,6 @@
 process FLYE {
     tag "$meta.id"
-    label 'process_high'
+    label 'process_high_memory'
```

Comment:
I am kinda curious, if you were to update this with `nf-core tools` does it update this back to `process_high`?

Reply:
mm probably yes. So I need to move this into a local nf-core-modified and remove this from my installed modules since I've made quite a few changes at this point.